src/js/component.js

```diff
@@ -692,23 +688,21 @@ Component.prototype.off = function(first, second, third){
  *     myComponent.one(otherElement, 'eventName', myFunc);
  *     myComponent.one(otherComponent, 'eventName', myFunc);
  *
- * @param  {String|vjs.Component}  first   The event type or other component
+ * @param  {String|Component}  first   The event type or other component
  * @param  {Function|String}       second  The listener function or event type
  * @param  {Function=}             third   The listener function for other component
- * @return {vjs.Component}
+ * @return {Component}
  */
 Component.prototype.one = function(first, second, third) {
-  var target, type, fn, thisComponent, newFunc;
-
   if (typeof first === 'string' || Lib.obj.isArray(first)) {
     Events.one(this.el_, first, Lib.bind(this, second));
   } else {
-    target = first;
-    type = second;
-    fn = Lib.bind(this, third);
-    thisComponent = this;
+    let target = first;
+    let type = second;
+    let fn = Lib.bind(this, third);
+    let thisComponent = this;
 
-    newFunc = function(){
+    let newFunc = ()=> {
```

Comment:
heh this is actually exactly what I assumed I would have to revert.

I'm not sorry.